docs/ABI.rst

```diff
@@ -1,1230 +0,0 @@
-:orphan:
-
-.. @raise litre.TestsAreMissing
-.. _ABI:
```

Comment:
`:orphan:` just means "does not show up in a table of contents". Since Markdown documents aren't automatically grouped across files, this is fine to just drop.